common/include/pcl/point_types_conversion.h

```diff
@@ -364,6 +364,24 @@ namespace pcl
       out.push_back (p);
     }
   }
+ 
+  /** \brief Convert a XYZHSV point cloud to a XYZRGB
+    * \param[in] in the input XYZHSV point cloud
+    * \param[out] out the output XYZRGB point cloud
+    */
+  inline void 
+  PointCloudXYZHSVtoXYZRGB (const PointCloud<PointXYZHSV>& in,
+                            PointCloud<PointXYZRGB>&       out)
+  {
+    out.width   = in.width;
+    out.height  = in.height;
+    for (const auto &point : in.points)
```

Comment:
Can we avoid using the .points field?

Reply:
Simply removing that and only writing `in` should work, right?

Reply:
The .points field is used in all other tranformation functions, i thought it was correct to use it.

Reply:
I made it only removing the .points file from all in variables. You were right mvieth.